fdbserver/include/fdbserver/DataDistributionTeam.h

```diff
@@ -69,9 +69,41 @@ FDB_DECLARE_BOOLEAN_PARAM(ForReadBalance);
 FDB_DECLARE_BOOLEAN_PARAM(PreferLowerReadUtil);
 FDB_DECLARE_BOOLEAN_PARAM(FindTeamByServers);
 
+class TeamSelect {
+public:
+	enum Value : int8_t {
+		INVALID = 0,
+		AVOIDMOVEMENT, // Avoid data movement by selecting srcTeam if it's not busy
+		WANTSRCSERVERS, // Ask for srcTeam
+		WANTTRUEBEST, // Ask for the most or least utilized team
```

Comment:
The names should probably have underscores to separate the words

Reply:
Done